NOTICE

```diff
@@ -7,3 +7,10 @@ The Apache Software Foundation (http://www.apache.org/).
 The initial code for the Polaris project was donated
 to the ASF by Snowflake Inc. (https://www.snowflake.com/) copyright 2024.
 
+--------------------------------------------------------------------------------
+
+This project includes code from Project Nessie, developed at Dremio,
```

Comment:
Would it make sense to mention that Nessie code is included in source form (copy of some code fragments)? That would help with reasoning about why only the short notice is included, I hope.

Reply:
That's "implicit" as we are in the `NOTICE` from the source distribution, so meaning covering only code copy.